go.mod

```diff
@@ -5,20 +5,23 @@ go 1.14
 require (
 	github.com/AlecAivazis/survey/v2 v2.2.2
 	github.com/Netflix/go-expect v0.0.0-20180615182759-c93bf25de8e8
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
```

Comment:
i think you should run `go mod tidy` in your project and see what updates here... just curious if `segmentio` needs to bring all these in or if something was included accidentally

Reply:
I did `go mod tidy`, but I didn't see any changes to commit